widgetssdk/src/main/java/com/glia/widgets/chat/controller/ChatController.kt

```diff
@@ -232,7 +232,7 @@ internal class ChatController(
         addFileAttachmentsObserverUseCase.execute(fileAttachmentObserver)
         minimizeHandler.addListener { minimizeView() }
         createNewTimerCallback()
-        callTimer.addFormattedValueListener(timerStatusListener)
+        callTimer.addFormattedValueListener(timerStatusListener!!) // Listener is created one line before
```

Comment:
I would try to avoid the force unwrap operation. If it's not possible to use `lateinit` or `by lazy`, I would use the `let {}`construction:
`timerStatusListener?.let{ callTimer.addFormattedValueListener(it) }`

Reply:
Missed it when fixing the other one.
It is fixed now in cdc877a